src/app/app-routing.module.ts

```diff
@@ -8,18 +8,18 @@ import { ItineraryViewContoller } from './components/itinerary/itineraryshow/iti
 import { NotFoundComponent } from './components/not-found/not-found.component';
 import { LoginComponent } from './components/operations/login/login.component';
 import { RegisterComponent } from './components/operations/register/register.component';
+import { ProfileComponent } from './components/profile/profile.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7182177)
```suggestion
import { ProfileComponent } from "./components/profile/profile.component";
```